### ISSUE TITLE
Difference to sokra sourcemap visualizer?

### ISSUE THREAD

**Issue by paulirish** [rbuckton/sourcemap-visualizer#4]
http://sokra.github.io/source-map-visualization/ is strikingly similar, including the example used here: http://i.imgur.com/YAQJqY5.png

Was the idea to port the essentials to TS and evolve it from there?

Just curious... would be good to add a note about the relationship to the readme.

**Comment by rbuckton** [comment]
@paulirish  I originally started this to build up a comprehensive parser for SourceMaps and IndexMaps, and added the visualization later. I've seen the UI from Sokra's visualizer and it was the inspiration for some of the UI on the output report, though I haven't looked at the internals so I'm not certain if there are any similarities between the implementations.  This is definitely not a port of Sokra's visualizer to TS, but I can add a note about the UI similarities if necessary.

**Comment by paulirish** [comment]
Yeah I think it'd be good. As a user I want to know if i'd be better off using this instead of the sokra one i'm slightly more familiar with.

"We share a consistent visual language to describe the mapping and relationship, but beyond visualizing, this project does… "  sort of thing.
